Frontend/src/data/rental/onSave.js

```diff
@@ -43,15 +47,14 @@ const updateItemStatus = async (item, status) => {
 export default async (context) => {
   const { doc, closePopup, createNew, contextVars } = context;
   setNumericValuesDefault0(doc, columns);
-
   // item changed, reset initial item to status available
   if (
     contextVars.initialItemId !== undefined &&
     contextVars.initialItemId !== doc.item_id
   ) {
     try {
       const initialItem = await fetchItemById(contextVars.initialItemId);
-      await updateItemStatus(initialItem, "instock");
+      await updateItemStatus(initialItem, "instock", undefined);
```

Comment:
not sure if this is necessary in JS

Reply:
No, you don't need the undefined here. If you only pass 2 arguments, the last one will be undefined anyway.